include/ignition/gui/qml/IgnSnackBar.qml

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.3
+import QtQuick.Window 2.2
+import QtQuick.Dialogs 1.0
+
+Popup {
+  id: snackbar
+  modal: duration == 0
+  focus: duration == 0
+  x: (window.width - width) / 2
+  y: window.height - window.height / 6
+  width: window.width - window.width / 6
+  contentHeight: notificationColumn.height
+
+  background: Rectangle {
+    border.color: "#444"
+    color: "#c9daf8"
```

Comment:
We should avoid using custom colors as much as possible and instead use colors from the current palette (`Material.primary`, `Material.accent`, `Material.background`). 

The [official spec](https://material.io/components/snackbars#anatomy) recommends a grey color, but I think QtQuick doesn't make it straight-forward to have that follow a dark / light theme. So I think we could stick to `Material.background` here.